src/main/java/se/sundsvall/emailreader/integration/graph/GraphClient.java

```diff
@@ -103,4 +104,51 @@ public void moveEmail(final String userId, final String messageId, final String
 			setUnHealthyConsumer.accept("[GRAPH] Could not move email from inbox");
 		}
 	}
+
+	private String findFolderId(final String userId, final String folderId, final Consumer<String> setUnHealthyConsumer) {
+		try {
+			final var result = graphServiceClient
+				.users()
+				.byUserId(userId)
+				.mailFolders()
+				.get();
+
+			if (result == null) {
+				return createFolder(userId, folderId, setUnHealthyConsumer);
```

Comment:
createFolder can return null (if something goes wrong).
So instead of throwing some kind of dedicate exception, we will throw a NPE if this happens.

Possible solutions are:

1. Re-trhow the caught exception from `createFolder`
2. Check for null values in this if-clause and throw an exception (in order to avoid a NPE)

(I think that option 1 would be the most straight forward)